mcs_benchmark_data/_benchmark_submission_transformer.py

```diff
@@ -29,44 +35,44 @@ def _uri_base(self):
     def _benchmark_score_classes(self):
         return {"TestScore": TestScore, "DevScore": DevScore}
 
-    def _transform(
+    def transform(
         self,
         *,
-        extracted_path: Path,
+        extracted_data_dir_path: Path,
         file_names: _BenchmarkFileNames,
-        submission_name: str,
         **kwds,
     ) -> Generator[_Model, None, None]:
 
-        submission_data_jsonl_file_path = extracted_path / getattr(
+        submission_data_jsonl_file_path = extracted_data_dir_path / getattr(
             file_names, "metadata"
         )
 
-        submission_jsonl_file_path = extracted_path / getattr(
-            file_names, "submission_file_name"
+        submission_jsonl_file_path = (

```

Comment:
Submissions are going to have different formats, this is premature reuse.

Reply:
Instead of dealing with the different formats you're pushing the actual transformation (from whatever the system emits to your "standard" format) outside the ETL pipeline into one-off code, when doing this transformation is exactly the job of the pipeline. It's better to keep everything in the pipeline organization.